### PR TITLE
Rename data modal struct and enum for xcframework

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Code update for https://github.com/wireapp/wire-ios-data-model/pull/1167

### CODE REVIEW

File: Sources/Helpers/ZMTBaseTests+CoreDataStack.swift [hunk 1]
```diff
@@ -43,8 +43,8 @@ extension ZMTBaseTest {
 
     @objc
     func setupCaches(in coreDataStack: CoreDataStack) {
-        let userImageCache = UserImageLocalCache()
-        let fileAssetCache = FileAssetCache()
+        let userImageCache = UserImageLocalCache(location: nil)
```
Comment (KaterinaWire) [inline Sources/Helpers/ZMTBaseTests+CoreDataStack.swift:46]: Why did you remove the default value of the location parameter in `init` method (in DM)?

**Review by KaterinaWire — APPROVED** [review]

**Review by typfel — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release major
